src/engine/enginemaster.h

```diff
@@ -66,7 +66,7 @@ class EngineMaster : public QObject, public AudioSource {
     // be called by SoundManager.
     const CSAMPLE* buffer(AudioOutput output) const;
 
-    ChannelHandleAndGroup registerChannelGroup(const QString& group) {
+    ChannelHandleAndGroup registerChannelGroup(const QString& group) const {
```

Comment:
Should this really be `const` if it modifies a member object?

Reply:
It doesn't modify anything?
I also thought about renaming this method, it basically just does a conversion, it is more of a getter rather than something that "registers" anything-

Reply:
Well, it does kinda modify as it might create a handle, though it is questionable how much that matters since the function is still idempotent as seen from outside the ChannelHandleFactory. Removed `const` for now.